File: AppGR/FontEnd/app/src/main/java/com/example/appgrit/models/SignInModel.java
package com.example.appgrit.models;

public class SignInModel {
    private String Email;
    private String Password;

    public SignInModel() {
    }

    public SignInModel(String email, String password) {
        Email = email;
        Password = password;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }
}

File: AppChat/BackEnd/AppChat/Models/SignInModel.cs
using System.ComponentModel.DataAnnotations;

namespace AppGrIT.Models
{
    public class SignInModel
    {
        [Required]
        [EmailAddress]
        public string Email { get; set; } = null!;

        [Required]
        [MinLength(6)]
        public string Password { get; set; } = null!;
    }
}
